ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -84,6 +85,8 @@ def llmobs_set_tags(
         if usage != {}:
             span.set_tag_str(METRICS, json.dumps(usage))
 
+        record_span_created(span)
```

Comment:
If we're using `LLMObsTraceProcessor.process_trace()` to record spans being created, this is unnecessary and probably duplicate since process_trace runs on all spans 😄